test/jquery.ime.test.fixtures.js

```diff
@@ -2334,6 +2334,8 @@ var palochkaVariants = {
 		tests: [
 			{ input: 'c', output: 'చ్', description: 'Telugu c' },
 			{ input: 'ch', output: 'చ్', description: 'Telugu ch' }
+			{ input: 'j~nu', output: 'జ్ఞు', description: 'Telugu j~nu' }
```

Comment:
This definitely cannot work because there is no comma at the end of this line.

Also, if I type `j~nu` in this version, I get జ్ఞఉ. If I type just `j~n`, then I get "జ్ఞ".